From 3be3529ff35ae363fec3063827b8ac0d2e50c48a Mon Sep 17 00:00:00 2001
From: Kenneth Moreland <morelandkd@ornl.gov>
Date: Thu, 10 Jun 2021 10:54:46 -0600
Subject: [PATCH] Export tbb interface as vtkm::tbb

The dependent exports should be placed in the vtkm namespace.
---
 CMake/VTKmDeviceAdapters.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMake/VTKmDeviceAdapters.cmake b/CMake/VTKmDeviceAdapters.cmake
index aae081681..dc892f91b 100644
--- a/CMake/VTKmDeviceAdapters.cmake
+++ b/CMake/VTKmDeviceAdapters.cmake
@@ -46,7 +46,7 @@ if(VTKm_ENABLE_TBB AND NOT TARGET vtkm::tbb)
   add_library(vtkmTBB INTERFACE)
   add_library(vtkm::tbb ALIAS vtkmTBB)
   target_link_libraries(vtkmTBB INTERFACE TBB::tbb)
-  set_target_properties(vtkmTBB PROPERTIES EXPORT_NAME tbb)
+  set_target_properties(vtkmTBB PROPERTIES EXPORT_NAME vtkm::tbb)
   install(TARGETS vtkmTBB EXPORT ${VTKm_EXPORT_NAME})
 endif()
 
-- 
GitLab

